adobe/react-spectrum
# Prevent menu flicker when closing ComboBox via blur

Closes https://github.com/adobe/react-spectrum/issues/2181

## ✅ Pull Request Checklist:


## 📝 Test Instructions:

1. Go to a combobox story and type in the field in such a way that you have a list that is much shorter than the original list
2. Click right below the menu so that it closes it
3. The menu shouldn't briefly flicker and display the whole item list. It should only render the last list before the menu was closed

## 🧢 Your Project:

RSP
